vnext/Microsoft.ReactNative/Modules/AlertModule.cpp

```diff
@@ -38,6 +38,9 @@ void Alert::ProcessPendingAlertRequests() noexcept {
   dialog.PrimaryButtonText(Microsoft::Common::Unicode::Utf8ToUtf16(args.buttonPositive));
   dialog.SecondaryButtonText(Microsoft::Common::Unicode::Utf8ToUtf16(args.buttonNegative));
   dialog.CloseButtonText(Microsoft::Common::Unicode::Utf8ToUtf16(args.buttonNeutral));
+  if (args.defaultButton >= 0 && args.defaultButton <= 3) {
+    dialog.DefaultButton(static_cast<xaml::Controls::ContentDialogButton>(args.defaultButton));
```

Comment:
Note this API was introduced in RS2, I don't think we need an API level check as I believe RS2 is the min target version already.